src/components/pages/Navigation.vue

```diff
@@ -105,22 +104,26 @@ header.theme--dark.v-sheet
 
 
 
-
-      <v-menu
-      offset-y
-      :close-on-content-click="false"
-      :nudge-width="350"
-      v-model="showCaseMenu"
+      <v-popover
+              v-model="showCaseMenu"
       >
-      <template v-slot:activator="{ on }">
-        <v-btn text v-on="on" v-if="caseSummary && caseSummary.name">
-          {{ caseSummary.name }}
-        </v-btn>
-      </template>
+        <button v-if="caseSummary && caseSummary.name"
+                offset-y
+        style="margin-bottom: 6px">          {{ caseSummary.name }}</button>
 
-        <v-card>
-        </v-card>
-      </v-menu>
+        <template slot="popover"
+
+        >
+          <div style="color:black; background:white; width: 245px; text-align: left; margin-right: 5px; font-family: Poppins, sans-serif; font-weight: normal; font-size: 15px">
```

Comment:
Should we have the same font color as other text on the page and a little smaller font? And maybe wrap the inline css into a class? 

```suggestion
          <div style="color:#6a6a6a; background:white; width: 245px; text-align: left; margin-right: 5px; font-family: Poppins, sans-serif; font-weight: normal; font-size: 14px; padding-right:15px">

```